test/acceptance/static.test.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import Counter from '../../src/'
```

Comment:
Simply `/` would reference the root of one's *file system*. There are workarounds, but they are a little too elaborate for this project, in my opinion.